### PR TITLE
[IMP] - add multi-company management to calendar event type

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: calendar_event_type_multi_company/models/calendar_event_type.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class CalendarEventType(models.Model):
+
+    _inherit = 'calendar.event.type'
+
+    company_id = fields.Many2one(
+        'res.company',
+        'Company',
+        default=lambda self: self.env.user.company_id,
```
Comment (sbidoul) [inline calendar_event_type_multi_company/models/calendar_event_type.py]: _company_default_get

File: calendar_event_type_multi_company/security/calendar_event_type.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="calendar_event_type_comp_rule" model="ir.rule">
+        <field name="name">Calendar Event Type multi company rule</field>
+        <field name="model_id" ref="calendar.model_calendar_event_type"/>
+        <field name="global" eval="True"/>
```
Comment (sbidoul) [inline calendar_event_type_multi_company/security/calendar_event_type.xml]: @sbejaoui I think global is a computed field.

**Review by gva-acsone — APPROVED** [review]
Functional test +1

**Review by RobinetDenisAcsone — APPROVED** [review]
LGTM

**Comment by sbejaoui** [comment]
@OCA/core-maintainers , This PR is ready to merge

**Pull request merged**